### PR TITLE
Added example on CRI parsing longlogs

### PR DESCRIPTION
**Pull request by juan-coralogix**
# Description

Just adds an example to Readme.
On how to fix LongLogs in CRI / containerd with concat.

# How Has This Been Tested?

Its a readme, no testing done. But the configuration added to the readme has been tested in a UseCase scenario.
# Checklist:
- [ ] I have updated the relevant Helm chart(s) version(s)
- [ ] I have updated the relevant component changelog(s)
- [X] This change does not affect any particular component (e.g. it's CI or docs change)

### CODE REVIEW

**Comment by nicolastakashi** [comment]
@juan-coralogix can you run `make fmt-docs` and push?

File: logs/fluentd/k8s-helm/http/README.md [hunk 1]
```diff
@@ -88,3 +88,32 @@ By default this chart installs additional dependent chart:
 ## Coralogix Fluentd Buffer Alert
 
 In order to create an alert on Fluentd buffer in Coralogix, please see [coralogix-alert doc](https://github.com/coralogix/telemetry-shippers/blob/master/logs/fluentd/docs/coralogix-alerts.md)
+
+
+## Log Logs: containerd / CRI partial logs
+
+If your application is generating logs longer than 16k you should notice that docker dirver is splitting the log in multiple messages.
```
Comment (matej-g) [inline logs/fluentd/k8s-helm/http/README.md:95]:
```suggestion
If your application is generating logs longer than 16k, you should note that the Docker driver will split the logs into multiple messages.
```

File: logs/fluentd/k8s-helm/http/README.md [hunk 2]
```diff
@@ -88,3 +88,32 @@ By default this chart installs additional dependent chart:
...
+To fix this we can use concat to fix this.
```
Comment (matej-g) [inline logs/fluentd/k8s-helm/http/README.md:96]:
```suggestion
To resolve this, you can use the `concat` filter.
```

File: logs/fluentd/k8s-helm/http/README.md [hunk 3]
```diff
@@ -88,3 +88,32 @@ By default this chart installs additional dependent chart:
...
+
+First lets make sure that in the override file, that you use to deploy the helm, has logtag as one of the regex group keys, just like this.
```
Comment (matej-g) [inline logs/fluentd/k8s-helm/http/README.md:98]:
```suggestion
First ensure that in the override file used to deploy the Helm chart has `logtag` as one of the regex group keys, like the following:
```

File: logs/fluentd/k8s-helm/http/README.md [hunk 4]
```diff
@@ -88,3 +88,32 @@ By default this chart installs additional dependent chart:
...
+```yaml
+<pattern>
+  format /^(?<time>[^ ]+) (?<stream>stdout|stderr) (?<logtag>[^ ]*) (?<message>.*)$/
+  time_format %Y-%m-%dT%H:%M:%S.%L%z
+  keep_time_key true
+</pattern>
+```
+If that is not the case please replace the existing one with this one.
+
+Then next to the source we will add the following filter that will concat the logs:
```
Comment (matej-g) [inline logs/fluentd/k8s-helm/http/README.md:109]:
```suggestion
Then, next to the source, you'll need to add the following filter that will concat the logs:
```

**Review by matej-g — COMMENTED** [review]
Looks good, some stylistic suggestions + @nicolastakashi already mentioned we should fix the failing CI

**Review by nicolastakashi — APPROVED** [review]

**Pull request merged**